container/Dockerfile.tensorrt_llm

```diff
@@ -224,6 +220,10 @@ COPY --from=ghcr.io/astral-sh/uv:latest /uv /uvx /bin/
 RUN mkdir /opt/dynamo && \
     uv venv /opt/dynamo/venv --python 3.12
 
+# Install genai-perf for benchmarking
+RUN . /opt/dynamo/venv/bin/activate && \
+	uv pip install genai-perf==$GENAI_PERF_VERSION
```

Comment:
FYI I touched similar part of the file here: https://github.com/ai-dynamo/dynamo/pull/1256